client/blocks/plan-storage/block.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import filesize from 'filesize';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from 'components/progress-bar';
+
+const ALERT_PERCENT = 80;
+const WARN_PERCENT = 60;
+
+class PlanStorageBlock extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		mediaStorage: PropTypes.object,
+		siteSlug: PropTypes.string.isRequired,
+		sitePlanName: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		siteSlug: ''
+	};
+
+	getPlanTranslation() {
+		const { sitePlanName, translate } = this.props;
+
+		if ( sitePlanName === 'Premium' ) {
+			return translate( 'Premium Plan', { context: 'Short plan name on storage block' } );
+		} else if ( sitePlanName === 'Free' ) {
+			return translate( 'Free Plan', { context: 'Short plan name on storage block' } );
+		}
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Premium Plan', { context: 'Short plan name on storage button'} )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/storage-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?